docs/reference/ml/df-analytics/apis/get-inference-trained-model.asciidoc

```diff
@@ -133,8 +137,86 @@ The license level of the trained model.
 
 `metadata`:::
 (object)
-An object containing metadata about the trained model. For example, models 
+An object containing metadata about the trained model. For example, models
 created by {dfanalytics} contain `analysis_config` and `input` objects.
+.Properties of metadata
```

Comment:
This isn't formatting properly, so I think a continuation character is required.
```suggestion
+
.Properties of metadata
```

Reply:
I'll fix this in https://github.com/elastic/elasticsearch/pull/62643